runtime/doc/options.txt

```diff
@@ -4245,6 +4245,20 @@ A jump table for the options with a short description can be found at |Q_op|.
 	Defines the maximum time in msec between two mouse clicks for the
 	second click to be recognized as a multi click.
 
+							*'msgfunc'* *'msgfu'*
+'msgfunc' 'msgfu'	string	(default: empty)
+			global
+	This option specifies a function to be called by |messages|.
+	It overwrites the default messages behavior.
```

Comment:
"It replaces" reads better than "It overwrites" if we're referring to a behavior

To be more precise, I'd reword this sentence as "It replaces the default messages' display functionality" as the behavior processing-wise remains the same, it just customizes how the messages are handled for the UI.